site/docs/hosting/vps.md

```diff
@@ -2,24 +2,124 @@
+   If you use your own Bot API server, choose the location closest to it.
+2. **Server type.**
+   Choose the option "Clean OS".
+3. **Operating system.**
+   We will use Ubuntu 22.04.
+   If you choose a different system, some steps may be different, so be careful.
+4. **Server name.**
+5. **Root password.**
+   Make a strong password and keep it in a safe place!
+6. **SSH key**.
+   Skip this step.
+   We will set up SSH keys [later](#ssh-keys).
+
+After the server is created, you can connect to it using SSH:
+
+> SSH (_Secure Shell_) is a network protocol for remote control of a computer (server) and file transfer.
 
-## systemd
+```sh
+ssh root@<ip-address>
+```
+
+replacing `<ip-address>` with the IP address of your server, which you can find on the server management page.
+
+::: tip Separate User for Each Application
+In this guide, all actions with the server will be performed as the root user.
+This is done on purpose to simplify the material.
+However, in reality, the root user should only be responsible for general services (web server, database, etc.), and applications should be run by individual non-root users.
+This approach ensures the security of confidential data and prevents hacking of the entire system.
+At the same time, it imposes some inconveniences.
+Describing all these points unnecessarily increases the complexity of the article, which we try to avoid.
+:::
+
+## Starting the Bot
+
+We now have a server at our disposal where we can run the bot to keep it running around the clock.
+
+> To simplify the beginning of the article, we skipped the step of delivering the code to the server automatically each time after pushing your code, but it is described [below](#ci-cd).
+>
+> For now, you can copy local files to a remote server using the following command:
+>
+> ```sh
+> scp <path-to-local-files> root@<ip-address>:<path-to-remote-directory>
+> ```
+>
+> replacing `<path-to-local-files>` with the path to local files, `<ip-address>` with the IP address of your server, and `<path-to-remote-directory>` with the path to the directory where the bot's source code should be stored on the server.
+
+:::tip Don't forget to install the runtime!
+To run the bot, you need to install Node.js or Deno on the server, depending on the runtime in which the bot will run.
+This is beyond the scope of this article, so you will need to do it yourself.
+You probably already did this when [getting started](../guide/getting-started.md), so you should be familiar with the steps. :wink:
+:::
+
+Below are two ways you can keep your bot running smoothly.
+
+### systemd
 
 systemd is a powerful service manager which is pre-installed on many Linux distributions, mainly Debian-based ones.
 
-### Getting the Start Command
+#### Getting the Start Command
 
 1. Get the full path of your runtime:
 
-```sh
-# If using Deno
+::: code-group
+
+```sh [Deno]
 which deno
+```
 
-# If using Node.js
+```sh [Node.js]
 which node
 ```
 
+:::
```

Comment:
Shouldn't this be indented in order to be part of the list entry? Currently, it interrupts the list.

Reply:
This makes sense (even Markdownlint complains about it). But then you lose the syntax highlighting in the code editor, which I don't like. I discussed this with Roj here https://t.me/c/1322374388/4249

Reply:
I'm aware, see https://t.me/c/1322374388/4265?thread=4249

Reply:
Wouldn't it be better to apply this change in this PR, since it is the same `ol-prefix` rule.
Then it will have to be changed here as well.

Reply:
Sure, go ahead

Reply:
done